Samples/ResourceManagement/cs-winforms-unpackaged/Program.cs

```diff
@@ -1,54 +1,60 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Curiously the whole file shows as changed in the diff on GitHub... I wonder if maybe the encoding of the file changed perhaps?

Reply:
VS asked if I wanted to fix line endings and I said yes. That may be why. In CF, only my additions show as new...

Reply:
Ah, from offline discussion what is happening is that the file is being changed from `LF` endings to `CR` `LF` endings.
Since CR LF is more common/expected on Windows, it is probably better to change this anyways...

Reply:
Keeping the new line endings per Jeff's reasoning.